apps/codelab/src/app/admin/feedback/feedback.component.css

```diff
@@ -0,0 +1,9 @@
+.panel {
+  margin-bottom: 10px;
+}
+
+.header .matinp {
```

Comment:
can we just style mat-form-field here? Or name this class in a nicer way?

Reply:
☝️